azurelinuxagent/common/event.py

```diff
@@ -208,7 +208,11 @@ def save_event(self, data):
             logger.warn("Cannot save event -- Event reporter is not initialized.")
             return
 
-        fileutil.mkdir(self.event_dir, mode=0o700)
+        try:
+            fileutil.mkdir(self.event_dir, mode=0o700)
+        except (IOError, OSError) as e:
+            msg = "Failed to create events folder {0}. Error: {1}".format(self.event_dir, ustr(e))
+            raise EventError(msg)
```

Comment:
The caller of this method handles `EventError` exceptions.

Reply:
In the caller
```
        try:
            self.save_event(json.dumps(data))
        except EventError as e:
            logger.error("{0}", e)
```

can you make the log statement periodic? thanks

Reply:
(intention is not to flood the serial console)

Reply:
Makes sense, updated to every 15 minutes.